### ISSUE TITLE
cargo-edit 0.1.3 does not build and install with rustc 1.15.0-nightly (ebeee0e27 2016-12-04)

### ISSUE THREAD

**Issue by brson** [rust-lang/rust#38444]
`rustc 1.15.0-nightly (ebeee0e27 2016-12-04)`

`cargo install cargo-edit`

```
   Compiling curl-sys v0.1.35
   Compiling lazy_static v0.2.2
   Compiling winapi-build v0.1.1
   Compiling toml v0.1.30
   Compiling quick-error v1.1.0
   Compiling kernel32-sys v0.2.2
   Compiling nom v1.2.4
   Compiling semver v0.2.3
   Compiling libc v0.2.18
   Compiling thread-id v2.0.0
   Compiling thread_local v0.2.7
   Compiling log v0.3.6
   Compiling memchr v0.1.11
   Compiling aho-corasick v0.5.3
   Compiling regex v0.1.80
   Compiling docopt v0.6.86
   Compiling unicode-bidi v0.2.3
^P   Compiling idna v0.1.0
   Compiling url v1.2.4
   Compiling curl v0.2.19
   Compiling cargo-edit v0.1.3
error: unnecessary qualification
  --> /home/ec2-user/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-edit-0.1.3/src/manifest.rs:13:1
   |
13 | quick_error! {
   | ^
   |
note: lint level defined here
  --> /home/ec2-user/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-edit-0.1.3/src/lib.rs:4:176
   |
4  | #![deny(missing_docs, missing_debug_implementations, missing_copy_implementations, trivial_casts, trivial_numeric_casts, unsafe_code, unstable_features, unused_import_braces, unused_qualifications)]
   |                                                                                                                                                                                ^^^^^^^^^^^^^^^^^^^^^
   = note: this error originates in a macro outside of the current crate

error: aborting due to previous error

error: failed to compile `cargo-edit v0.1.3`, intermediate artifacts can be found at `/tmp/cargo-install.2veDEfppY1Ba`

Caused by:
  Could not compile `cargo-edit`.
```

**Comment by brson** [comment]
cc @killercup

**Comment by killercup** [comment]
Interesting. I'm not sure which `unused_qualifications` is triggered in the macro code, and don't have the time to investigate this now. Do you think this is a bug in rustc or something that was actually just not caught before and is fixed now?

I can release a hotfix release without that lint in the meantime.

**Comment by arielb1** [comment]
1.15.0 is now beta

**Comment by killercup** [comment]
Just published cargo-edit 0.1.4 without this lint (and a bunch of dependency updates).

**Comment by brson** [comment]
Since this is just a change in lints, I don't think there's anything to be done.
